src/components/pill-selector-dropdown/PillSelector.js

```diff
@@ -26,12 +26,13 @@ type Props = {
     getPillClassName?: (option: Option) => string,
     /** Function to retrieve the image URL associated with a pill */
     getPillImageUrl?: (data: { id: string | number, [key: string]: any }) => string,
+    innerRef?: React.Ref<any>,
     inputProps: Object,
     onInput: Function,
     onRemove: Function,
     onSuggestedPillAdd?: Function,
     placeholder: string,
-    selectedOptions: Array<Object> | List<Object>,
```

Comment:
why is this removed?

Reply:
More specifically, only Array<Object> was being removed. Flow was complaining of ambiguity of the type of the value assigned to selectedOptions in defaultProps. 

```
Could not decide which case to select. Since case 1 [1] may work but if it doesn't case 2 [2] looks promising too. To
fix add a type annotation to unknown element type of empty array [3].

 [1][2] 57│     selectedOptions: Array<Object> | List<Object>,
          :
        67│         error: '',
        68│         inputProps: {},
        69│         placeholder: '',
    [3] 70│         selectedOptions: [],
        71│         validator: () => true,
        72│     };
        73│

```

Additionally, there are functions called on selectedOptions that are only supported in List and not in Array.

```
Property size is missing in array type [1].

 [1] 35│     selectedOptions: Array<Object>,
       :
     79│     getNumSelected = (): number => {
     80│         const { selectedOptions } = this.props;
     81│
     82│         return typeof selectedOptions.size === 'number' ? selectedOptions.size : selectedOptions.length;
     83│     };
     84│
     85│     getPillsByKey = (key: string): Array<any> => {

```